packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/Content.tsx

```diff
@@ -90,6 +99,13 @@ const UserAccountMenu = ({
             ></Trans>
           </p>
         )}
+
+        {isProvider && (
+          <p className={`${style.ssoBadge} oui-badge oui-badge_warning`} data-navi-id="account-email">
```

Comment:
**nitpick:** Same navi-id `account-email` is used twice. Could you please rename it?

cc @fchabrolle

Reply:
Done